client/src/main/java/io/pravega/client/ClientConfig.java

```diff
@@ -30,8 +29,7 @@
      2. pravega://ip1:port1,ip2:port2,...
         This is used to autodiscovery the controller endpoints from an initial controller list.
     */
-    @Builder.Default
-    private final URI controllerURI = URI.create("tcp://localhost");
+    private URI controllerURI = URI.create("tcp://localhost");
```

Comment:
Please move this assignment to an else block in the constructor so this field can be final.

Reply:
Dropping the constructor. Using `toBuilder` lombok functionality to fix the issue.